### PR TITLE
examples: Examples of .htaccess and README.md

### PR DESCRIPTION
**Pull request by bact**
* Add `examples/README.md` with examples of .htaccess and links to examples of README.md and examples/ontology/ - as proposed in #1530 and #1643
* Add `examples/ontology` that redirects to `https://w3id.org/example` - as proposed in #1643

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+### Put ID info and maintainer info inside .htaccess
+
+You are encouraged to put a breif ID info and a maintainer info in the comment
+(lines staring with `#` character) of a `.htaccess` file.
+
+```ApacheConf
+# Example
+#
+# https://w3id.org/example redirects to https://example.com/
```
Comment (dgarijo) [inline examples/README.md:39]: This is not correct.

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+GitHub will automatically display the content of a `README.md` to repository visitors.
+
+An example of a good README file: [w3id.org/dggs/README.md](https://github.com/perma-id/w3id.org/blob/master/dggs/README.md)
+
+
+## Publish vocabularies with W3ID
+
+If you plan to publish a vocabulary/ontology with W3ID,
+see https://w3id.org/examples/ontology.
```
Comment (dgarijo) [inline examples/README.md:191]: Wouldn't it be better to link to /example/ directly? The one in /examples is a redirect.

**Comment by dgarijo** [comment]
@bact 
I added small comments. Also, aren't you missing the `.htaccess` file redirecting to example.org in the /examples?

File: examples/README.md (hunk 2 above)
Comment (bact) [reply examples/README.md]: Thank you. That's better. I have changed that accordingly.

File: examples/README.md (hunk 1 above)
Comment (bact) [reply examples/README.md]: Thank you. I have fixed all the instances that mentioned https://w3id.org/example wrongly.

In that example, I replace it with w3id.org/examples/simple instead.

For .htaccess in examples/, I am not sure where to redirect it to.

**Comment by bact** [comment]
> @bact I added small comments. Also, aren't you missing the `.htaccess` file redirecting to example.org in the /examples?

For .htaccess in examples/, I am not sure where to redirect it to.

**Comment by bact** [comment]
I'm also not sure about what is the best practice on HTTP status, which one is recommended for W3ID use case: 302, 303, or 307? Or it really depends on each ID use case?

**Comment by dgarijo** [comment]
I think it depends on the use case. 303 is the recommended for semantic resources at least.

**Comment by bact** [comment]
> > @bact I added small comments. Also, aren't you missing the `.htaccess` file redirecting to example.org in the /examples?
> 
> For .htaccess in examples/, I am not sure where to redirect it to.

decided to redirect it to https://github.com/perma-id/w3id.org/tree/master/examples - see #3601

**Pull request merged**